v2/pkg/operator/collector.go

```diff
@@ -99,16 +101,16 @@ func (o *Collector) OperatorImageCollector(ctx context.Context) ([]v1alpha3.Copy
 			if err != nil {
 				return []v1alpha3.CopyImageSchema{}, err
 			}
-
 			//read the link to the manifest
 			if len(oci.Manifests) == 0 {
 				return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] no manifests found for %s ", op.Catalog)
-			} else {
-				if !strings.Contains(oci.Manifests[0].Digest, "sha256") {
-					return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] the disgets seems to incorrect for %s ", op.Catalog)
-				}
 			}
-			manifest := strings.Split(oci.Manifests[0].Digest, ":")[1]
+			validDigest, err := digest.Parse(oci.Manifests[0].Digest)
+			if err != nil {
+				return []v1alpha3.CopyImageSchema{}, fmt.Errorf("[OperatorImageCollector] the disgets seems to be incorrect for %s ", op.Catalog)
+			}
+
+			manifest := validDigest.Hex()
```

Comment:
Hi @sherine-k,

It seems the Hex() func is deprecated. It is saying to use Digest.Encoded instead